containers/verifier/Dockerfile.j2

```diff
@@ -0,0 +1,26 @@
+FROM golemfactory/blender:{{ golemfactory_blender_version }}
+
+# Add dependencies from golem verification and concent
+COPY testing.pref                             /etc/apt/preferences.d/
+COPY golem/imgverifier-requirements.txt        /golem/work/
```

Comment:
Please keep it aligned with the other lines.

Reply:
Done.